### PR TITLE
Updated docs for 4.7

### PR DESCRIPTION
**Pull request by reginapizza**
**What type of PR is this?**

/kind documentation

**What does this PR do / why we need it**:

Updates docs for 4.7 clusters. Some previous images/instructions were out of date. 

Done:
 - removed some files and images for installation of pipelines/argo cd operators
 - added some new images to reflect the way things currently look in 4.7
 - removed mentions of 4.6
 - changed any mention of Application Stages to Environments
 - changed "ArgoCD" to "Argo CD" (only for docs, not for actual code)
 - fixed error in filename for `<Quay user>-robot-auth.json`

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [x] Documentation has been updated.

**Which issue(s) this PR fixes**:

No issue in kam repo for this but tracked in Jira as [GITOPS 545](https://issues.redhat.com/browse/GITOPS-545)

### CODE REVIEW

**Comment by amitkrout** [comment]
> @reginapizza Docs are out of date. Could you please run `make cmd-docs`?

 + make cmd-docs
go run tools/cmd-docs/main.go
++ git status -s
+ [[ ! -z  M docs/commands/kam_build.md ]]
+ echo 'command-documentation is out of date (run make cmd-docs).'
command-documentation is out of date (run make cmd-docs).
+ exit 1 

Test log - https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-developer_kam/190/pull-ci-redhat-developer-kam-master-unit/1369148118699872256#1:build-log.txt%3A289

File: docs/FAQ/GitopsFAQ.md [hunk 1]
```diff
@@ -88,9 +88,9 @@ Also, if you're using the optional _commit-status-tracker_ controller, please se
 
 https://github.com/bitnami-labs/sealed-secrets#how-can-i-do-a-backup-of-my-sealedsecrets
 
-## ArgoCD is failing with forbidden: User "system:serviceaccount:openshift-gitops:argocd-application-controller" cannot create resource in API group
+## Argo CD is failing with forbidden: User "system:serviceaccount:openshift-gitops:argocd-application-controller" cannot create resource in API group
 
-The ArgoCD operator doesn't setup ArgoCD with enough permissions, this is a
+The Argo CD operator doesn't setup Argo CD with enough permissions, this is a
```
Comment (wtam2018) [inline docs/FAQ/GitopsFAQ.md:93]: @reginapizza @chetan-rns I don't think we need this workaround.

Comment (chetan-rns) [reply docs/FAQ/GitopsFAQ.md:93]: Yeah. It's no longer required

File: docs/devconsole/README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ $ kubectl create namespace pipelines-{console_username}-github
 $ kubectl create secret -n pipelines-{console_username}-github generic {console_username}-github-token --from-literal=token={user-token}
 ```
 
-6. Application Stages should now be populated with the list of applications.
+6. Environments should now be populated with the list of applications.
 
 ![Application_List](./img/GitOps_Applications.jpg)
```
Comment (chetan-rns) [inline docs/devconsole/README.md:31]: This image has `Application Stages`. Can we replace it with an equivalent `Environments` image?

**Review by chetan-rns — COMMENTED** [review]
Thanks for the PR! There are few images under `kam/docs/devconsole/img` that are not relevant to 4.7. Can we please get rid of them if they're no longer used? Everything else looks good to me :+1:

File: docs/devconsole/README.md (hunk 2 above)
Comment (reginapizza) [reply docs/devconsole/README.md:31]: Sorry, must have grabbed the wrong image for that. All good now and deleted the old pictures there as well.

**Review by chetan-rns — APPROVED** [review]
/lgtm
/approve

**Pull request merged**